spec/forms/legal_aid_applications/percentage_home_form_spec.rb

```diff
@@ -16,7 +16,7 @@
 
   describe '#save' do
     it 'updates application.percentage_home' do
-      expect { subject.save }.to change { application.reload.percentage_home.to_s.to_d }.to(percentage_home.to_d)
+      expect { subject.save }.to change { application.reload.percentage_home }.to(percentage_home)
```

Comment:
I think these `.to_s` and `.to_d` were introduced because this test was intermittently failing so I would leave them if I were you.